internal/controller/controller.go

```diff
@@ -420,10 +439,11 @@ func (f *PidFanController) calculateTargetPwm() int {
 
 	// map the target value to the possible range of this fan
 	maxPwm := fan.GetMaxPwm()
-	minPwm := fan.GetMinPwm() + f.minPwmOffset
+	// minPwm := fan.GetMinPwm() + f.minPwmOffset
```

Comment:
The `f.minPwmOffset` property is used to make sure a "neverStop" constraint is actually met. If a fan stops spinning when at minPwm, the `minPwmOffset` should be increased gradually until the fan is spinning again.

Reply:
I have reverted this change.